src/pages/Bridge/BridgeOverview/components/IssueForm/IssueForm.tsx

```diff
@@ -52,6 +58,10 @@ const IssueForm = ({ requestLimits, dustValue, issueFee }: IssueFormProps): JSX.
 
   useDebounce(() => setDebouncedAmount(amount), 500, [amount]);
 
+  const [securityDeposit, setSecurityDeposit] = useState<MonetaryAmount<CurrencyExt>>(
```

Comment:
@peterslany moved the state to the top of the component. This is something I should have commented when I reviewed your code, but normally we try to keep the states on the top of the component.